app/views/email_authentications/invitations/_facility_access_tree.html.erb

```diff
@@ -1,18 +1,18 @@
 <ul class="access-item-wrapper facility <%= "collapsed" unless tree_depth.zero? %>">
-  <% tree.each do |facility, _facility_info| %>
+  <% tree.each do |facility| %>
     <li>
       <div class="access-item facility <%= "collapsed" if root == :organization or root == :facility_group %>">
         <% tree_depth.times do %>
           <span class="spacer"></span>
         <% end %>
 
         <%= access_checkbox(
-                form,
                 :facilities,
                 facility,
                 page: page,
-                checked_fn: -> { user_being_edited.visible?(:facility, facility) }
+                checked_fn: -> { true }
```

Comment:
Why are the boxes always checked? Is this intentional?

Reply:
It is, but I understand how this isn't obvious. The `checked_fn` implicitly checks if the page is `edit?` and hence when this page is called for edits, the admin in question is the editee which is pre-filtered by whichever facilities they can see anyway, so they should all be checked.

Let me find a way to make this more obvious.

Reply:
What a great catch, this was in fact a regression I introduced, it should now be,

```
user_being_edited.visible_facilities.include?(facility)
```

...which makes sense.